### PR TITLE
Improve explore rooms when no results are found

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/17360

image
image

### CODE REVIEW

File: src/components/structures/RoomDirectory.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /*
-Copyright 2015, 2016 OpenMarket Ltd
 Copyright 2019 Michael Telatynski <7t3chguy@gmail.com>
-Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
+Copyright 2015, 2016, 2019, 2020 The Matrix.org Foundation C.I.C.
```
Comment (jryans) [inline src/components/structures/RoomDirectory.tsx:3]: Add 2021 somehow?

**Comment by niquewoodhouse** [comment]
> you need an invite to see them.

Just to check, is that true? I assumed it was but if you have an invite they will appear in the results?

**Comment by t3chguy** [comment]
If you have an invite it will appear in your room list invited section, the server likely will not return it here, but that's up to the server.

**Comment by niquewoodhouse** [comment]
Thanks, can we change the copy to just be "to join them" not "to see them" in that case, please?

**Review by niquewoodhouse — APPROVED** [review]

File: res/css/structures/_RoomDirectory.scss [hunk 2]
```diff
@@ -61,6 +61,39 @@ limitations under the License.
...
+            line-height: $font-20px;
+            color: $secondary-fg-color;
+            max-width: 464px; // easier reading
+        }
+
+        > hr {
+            margin: 0;
+            border: none;
+            height: 1px;
+            background-color: $header-panel-bg-color;
```
[the remark this answers is not in the document]
Comment (t3chguy) [reply res/css/structures/_RoomDirectory.scss:89]: @niquewoodhouse can you weigh in here, the lightness of the line matches the Figma

**Pull request merged**